Docs changes to reflect new assessment settings API and naming (#119)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/workspaces.mdx

| `data.attributes.vcs-repo.tags-regex`         | string  | (nothing) | A regular expression used to match Git tags. Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression.                                                                                                                                                                                                                                                                                                                      |
| `data.attributes.working-directory`           | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.                                                                                                                                                                                                                                                                                   |

> | `data.attributes.assessments-enabled`         | boolean | `false` | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `enforce-assessments`. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |

Reviewer: I'm trying to do complete sentences. Do you hate it? if you hate it put it back, but keep my spelling fix :-) hmmm, I also know what we're trying to do is warn folks that this used to be something else, but statements like "previously" can get stale. Like, when?!  I think we should remove that and just loudly tell folks everywhere else (blogs, idk other communications) that this happened.

The one place I think it makes sense to say "used to be drift detection" is on that page, because we renamed it, and I think it'd be weird to not mention that there. Push back on me if you completely disagree, but I just want to limit our use of things like this to only where they are very necessary.

```suggestion
| `data.attributes.assessments-enabled`         | boolean | `false` | Whether or not Terraform Cloud performs health assessments for the workspace. This property can be overridden by the organization setting `enforce-assessments`. This property is only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). This property is not available in Terraform Enterprise. |
```

_Without a VCS repository_

> **Note:** The `assessments-enabled` property is only acceppted by or returned from Terraform Cloud.

Reviewer: ```suggestion
**Note:** Only Terraform Cloud accepts and returns the `assessments-enabled` property. Health assessments are not available in Terraform Enterprise.
```

      "vcs-repo-identifier": null,
      "working-directory": null,
      "workspace-kpis-runs-count": null,

>       "assessments-enabled": false

Reviewer: Again, I just think we need to mention that certain properties aren't available in TFE everywhere we're going to show them (refer to my comment above about this)

_With a VCS repository_

> **Note:** The `assessments-enabled` property is only acceppted by or returned from Terraform Cloud.

Reviewer: ```suggestion
**Note:** Only Terraform Cloud accepts and returns the `assessments-enabled` property. Health assessments are not available in Terraform Enterprise.
```

| `data.attributes.vcs-repo.tags-regex`         | string         | (previous value) | A regular expression used to match Git tags. Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression.                                                                                                                                                                                                  |
| `data.attributes.working-directory`           | string         | (previous value) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.                                                                                                                                                               |

> | `data.attributes.assessments-enabled`         | boolean | `false` | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `enforce-assessments`. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |

Reviewer: ```suggestion
| `data.attributes.assessments-enabled`         | boolean | `false` | Whether or not Terraform Cloud performs health assessments for the workspace. This property can be overridden by the organization setting `enforce-assessments`. This property is only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). This property is not available in Terraform Enterprise. |
```
